buildSrc/src/main/groovy/local.java.vro-dep-constraints.gradle

```diff
@@ -0,0 +1,23 @@
+/**
+ * This plugin is used to specify versions of transitive dependencies to address security vulnerability alerts.
+ * If more formality is desired, convert this into a Platform -- https://docs.gradle.org/current/userguide/platforms.html#sub:platforms-vs-catalog
+ *   - This would result in creating another Gradle module/subfolder, which seems unnecessary at this point.
+ */
+plugins {
+    id 'java'
+}
+
```

Comment:
What is the naming convention for these files? local.java vs local.std.java. ?

Reply:
See `buildSrc/README.md`:

> The groups are identified as starter scripts, and roughly grouped:
> 
> * java - basic configuration around a typical build objective
> * metrics - build timing configuration
> * std - top-level configs based on target type
> 
> The `std` level of configuration is organized by artifact type.
> This is the only level meant to organize or include other configs.
> The design is meant to be easily overridden by teams that wish to depart from the standard configs.
> Teams are able to just copy the `std` level conventions to their project and override those they wish to change.
> Items they don't need to change can continue to be referred in their local configs.